sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/AnomalyDetectionConfigurationLiveTests.cs

```diff
@@ -204,7 +203,7 @@ public async Task CreateAndGetDetectionConfigurationWithSeriesGroupConditions()
 
             Assert.That(createdGroupConditions1, Is.Not.Null);
 
-            ValidateTempDataFeedDimensionKey(createdGroupConditions1.SeriesGroupKey, "Koltaka");
+            ValidateTempDataFeedDimensionKey(createdGroupConditions1.SeriesGroupKey, "Kolkata");
```

Comment:
Realized I had mistakenly typed the city name as "Koltaka" in some tests. We have some re-recordings because of this fix.